assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -298,6 +277,12 @@ function MediaPane(props) {
     [hasMore, isMediaLoading, isMediaLoaded, setNextPage]
   );
 
+  const onSearch = (v) => setSearchTerm({ searchTerm: v });
+
+  const incrementalSearchDebounceMedia = useFeature(
+    'incrementalSearchDebounceMedia'
```

Comment:
For global visibility, shall we have those with a little description in a separate file as const?

Reply:
added to flags.js